app/controllers/distributors_controller.rb

```diff
@@ -13,6 +13,12 @@ def show
     json_response(@distributor)
   end
 
+  # PUT /distributors/:id
+  def update
+    @distributor.update(update_params)
+    json_response(@distributor.as_json)
```

Comment:
no need for as_json since json_response converts it to a json already

Reply:
Love learning new things about backend!